snuba/query/allocation_policies/__init__.py

```diff
@@ -164,12 +166,45 @@ class AllocationPolicy(ABC, metaclass=RegisteredClass):
         >>>     QueryResultOrError(result=result)
         >>> )
 
-    The allocation policy base class has two public methods:
+    The allocation policy base class has two public methods for working with the actual quota:
         * get_quota_allowance
         * update_quota_balance
 
     These functions can be used to modify the behaviour of ALL allocation policies, use with care.
 
+    Configurations
+    --------------
+    The Allocation Policy base class comes with an abstraction on top of the runtime config which is used specifically
```

Comment:
* Remove the words that are not strictly necessary for the user.
* Since we're breaking from the existing patterns in the codebase, that should be stated
* examples > text